docs/package.json

```diff
@@ -28,14 +28,14 @@
     "@emotion/server": "^11.11.0",
     "@emotion/styled": "^11.13.0",
     "@mui/docs": "6.0.0-beta.4",
-    "@mui/icons-material": "^5.16.5",
+    "@mui/icons-material": "next",
     "@mui/joy": "^5.0.0-beta.48",
     "@mui/lab": "^5.0.0-alpha.173",
-    "@mui/material": "^5.16.5",
+    "@mui/material": "next",
```

Comment:
With Material UI v6 being used in the docs, I see TS errors like these:

```
src/modules/components/overview/InfoCard.tsx:52:50 - error TS7053: Element implicitly has an 'any' type because expression of type '50' can't be used to index type 'PaletteColor & PaletteColorChannel'.
  Property '50' does not exist on type 'PaletteColor & PaletteColorChannel'.

52             boxShadow: `0px 2px 30px 0px ${alpha(theme.palette.primary[50], 0.3)} inset, 0px 1px 6px 0px ${theme.palette.primary[100]}`,
                                                    ~~~~~~~~~~~~~~~~~~~~~~~~~
```

The branding theme overrides are located in `mui-docs`: https://github.com/mui/material-ui/blob/d9d0a37d6e11b83b3fefa716915cb1161a7a91db/packages/mui-docs/src/branding/brandingTheme.ts#L16-L17

How do we make MUI X aware of these overrides?
I tried adding `"../node_modules/@mui/monorepo/packages/mui-docs/src/*"` to `docs/tsconfig.json`'s `includes`, but it didn't work.
Any ideas? @LukasTy @Janpot

Reply:
Can you try upgrading the dependencies in the root package.json as well?

```diff
diff --git a/package.json b/package.json
index c7305d54b..cf9883d26 100644
--- a/package.json
+++ b/package.json
@@ -88,10 +88,10 @@
     "@emotion/cache": "^11.13.1",
     "@emotion/react": "^11.13.0",
     "@emotion/styled": "^11.13.0",
-    "@mui/icons-material": "^5.16.5",
+    "@mui/icons-material": "next",
     "@mui/internal-markdown": "^1.0.9",
     "@mui/internal-test-utils": "https://pkg.csb.dev/mui/material-ui/commit/225876f7/@mui/internal-test-utils",
-    "@mui/material": "^5.16.5",
+    "@mui/material": "next",
     "@mui/monorepo": "github:mui/material-ui#4a82b6b0e0395db8fa0a0d49b6b76de4516b1579",
     "@mui/utils": "^5.16.5",
     "@next/eslint-plugin-next": "14.2.5",
```